dotcom-rendering/src/model/article-schema.json

```diff
@@ -4595,16 +4595,16 @@
         "BadgeType": {
             "type": "object",
             "properties": {
-                "seriesTag": {
+                "imageSrc": {
                     "type": "string"
                 },
-                "imageUrl": {
+                "href": {
                     "type": "string"
                 }
             },
             "required": [
-                "imageUrl",
-                "seriesTag"
+                "href",
+                "imageSrc"
             ]
```

Comment:
What's the reason for this change?

Reply:
Series tag isn't always a series tag, for paid content it's a link to the sponsor's page. So href felt more appropriate, since that value is used as the href for the link. 

I guess imageURL could have stayed the same but I thought imageScr is more descriptive and since I was changing the other thing.. But I could be easily persuaded to change that back

Reply:
Okay! Is this something that needs to be updated in Frontend so the data comes through with the correct keys - or were they already set as both anyway?

Reply:
was always being set 👍